src/main/webapp/app/exercises/text/participate/text-result/text-result.component.html

```diff
@@ -5,11 +5,19 @@
             <fa-icon *ngFor="let i of repeatForEachCredit(block)" [icon]="block.icon"></fa-icon>
         </span>
         <div *ngIf="block.feedback && block.feedback.isSubsequent == undefined" class="feedback-text mt-1" [ngClass]="block.feedbackCssClass">
-            <b><fa-icon *ngIf="block.circleIcon" [icon]="block.circleIcon"></fa-icon>&nbsp;<span jhiTranslate="artemisApp.assessment.detail.feedback">Feedback</span>:</b>
+            <b
+                ><fa-icon *ngIf="block.circleIcon" [icon]="block.circleIcon"></fa-icon>&nbsp;<u
+                    ><span>{{ creditsTranslationForTextResultBlock(block) }}</span></u
+                >:</b
+            >
             <span [innerHTML]="buildFeedbackTextForReview(block.feedback)" id="text-feedback-{{ i }}"></span>
         </div>
         <div *ngIf="block.feedback && block.feedback.isSubsequent" class="feedback-text mt-1 alert alert-secondary">
-            <b><fa-icon *ngIf="block.circleIcon" [icon]="block.circleIcon"></fa-icon>&nbsp;<span jhiTranslate="artemisApp.assessment.detail.feedback">Feedback</span>:</b>
+            <b
+                ><fa-icon *ngIf="block.circleIcon" [icon]="block.circleIcon"></fa-icon>&nbsp;<u
+                    ><span>{{ creditsTranslationForTextResultBlock(block) }}</span></u
+                >:</b
+            >
```

Comment:
I have a few thoughts about this, let me know what you think :)

- I'd suggest to not underline it. I tried clicking / hovering on it! :D
From the spec:
```The default rendering of the <u> element in visual presentations clashes with the conventional rendering of hyperlinks (underlining). Authors are encouraged to avoid using the <u> element where it could be confused for a hyperlink.```
- The indentation and newline use of these lines look a bit weird :D can we fix that? E.g. currently there is a newline between ``<b`` and ``>``, for example :)

e.g. we could just go with
```html
<b>
    <fa-icon>...</fa-icon>
    &nbsp; {{ creditsTranslationForTextResultBlock(block) }}:
</b>
```

Reply:
Thanks for the review! I implemented your suggestions.